### PR TITLE
fix TOC issue

### PR DESCRIPTION
**Pull request by mairaw**
@shirhatti please take a look

What was your expectation for https://review.docs.microsoft.com/en-us/iis/web-development-reference/native-code-api-reference/native-code-api-reference?branch=master? I couldn't understand what was broken to fix that.

### CODE REVIEW

File: iis/web-development-reference/native-code-development-overview/TOC.md [hunk 1]
```diff
@@ -1,9 +0,0 @@
-# [Native-Code Development Overview](native-code-development-overview.md)
```
Comment (shirhatti) [inline iis/web-development-reference/native-code-development-overview/TOC.md]: @mairaw Can you refresh my memory here? Why was this file deleted? It seems like it's a useful addition to the TOC

**Comment by mairaw** [comment]
Need a fresh build

File: iis/web-development-reference/native-code-development-overview/TOC.md (hunk 1 above)
Comment (mairaw) [reply iis/web-development-reference/native-code-development-overview/TOC.md]: It was something you asked me. Let me rebuild and see what this PR changes.

Comment (mairaw) [reply iis/web-development-reference/native-code-development-overview/TOC.md]: It seems the TOC was already part of the main TOC, so this was duplicate.

Compare:
https://review.docs.microsoft.com/en-us/iis/web-development-reference/native-code-development-overview/native-code-development-overview?branch=pr-en-us-324

vs. 

clicking on any of these links on the main TOC
image

bring you to a sub-TOC
https://docs.microsoft.com/en-us/iis/web-development-reference/native-code-development-overview/native-code-development-overview
